test/e2e/syncer/syncer_test.go

```diff
@@ -153,14 +154,28 @@ func TestSyncerLifecycle(t *testing.T) {
 		}
 		if err != nil {
 			t.Errorf("saw an error waiting for downstream deployment %s/%s to be created: %v", downstreamNamespaceName, upstreamDeployment.Name, err)
-			return false
-		}
-		// Check for available replicas if downstream is capable of actually running the deployment
-		if len(framework.TestConfig.PClusterKubeconfig()) > 0 && expectedAvailableReplicas != deployment.Status.AvailableReplicas {
-			return false
 		}
 		return true
 	}, wait.ForeverTestTimeout, time.Millisecond*100, "downstream deployment %s/%s was not synced", downstreamNamespaceName, upstreamDeployment.Name)
 
+	t.Logf("Check for available replicas if downstream is capable of actually running the deployment")
+	expectedAvailableReplicas := int32(1)
+	require.Eventually(t, func() bool {
```

Comment:
Should we skip all of this if len(framework.TestConfig.PClusterKubeconfig()) is 0?

Reply:
done